docs/create-your-own-dapp.md

```diff
@@ -1,5 +1,21 @@
 # Creating Your First nOS dApp
 
+### Pre-requisites:
+
+First of all you'll need to build nOS browser locally, follow these instructions here:
+
+```
+// Install dependencies and launch the nOS client
+// This is used to start developing on the nOS client
+yarn install && yarn start
+
+// or if you want to create a temporary binary
+
+// Distribution command
+// After running `yarn dist` you can navigate to `dist` directory and execute the `nOS` binary in a terminal.
+yarn install && yarn dist
+```
+
```

Comment:
The @nosplatform/api-functions NPM package might be a good addition to the prerequisites.

Reply:
Hi @jeroenptrs thank you for the comments. 
Didn't know that NPM package existed for nos. Can you point me to it?

Reply:
The nOS api-functions are meant for the dApp within nOS, not nOS itself :D

Reply:
@DalderupMaurice correct. @ansrivas sorry for not being clear, the NPM package is used in your dApp, not the client. It's baked into the starter kit, but if you want to make a dApp from scratch (without the starter kit templates) you'll probably need the package.